dvc/repo/experiments/executor/base.py

```diff
@@ -701,13 +689,30 @@ def _auto_push(
     @classmethod
     def commit(
```

Comment:
`commit()` might not be a good place to `auto_push`.

Reply:
@skshetry Do you want to take it over?

Reply:
yeah, I have assigned it myself, and will get to it.